### ISSUE TITLE
Mod causing major lag

### ISSUE THREAD

**Issue by Linrox** [mrhid6/SF_Mod_RefinedPower#324]
Satisfactory Ver: CL#202470
Refined Power Ver:3.1.0
When Refined Power is installed and active i can't use the vanilla display screens, as it cause major lag, from 50-60fps down to 4-8fps.
If i remove the mod in the same save the lag disappears.
No other mods were active during the tests.

**Comment by mrhid6** [comment]
Update to 3.1.5

**Comment by Linrox** [comment]
Using the satisfactory mod manager doesn't show any updates available

**Comment by Linrox** [comment]
Opps. Found it.

**Comment by Linrox** [comment]
I will retest once update complete. Thanks

**Comment by Linrox** [comment]
Ok just retested with updated mods. lag still occurs

**Comment by mrhid6** [comment]
Please can you attach the save file to this issue?

**Comment by Linrox** [comment]
How do i attach the save file?

**Comment by mrhid6** [comment]
Drag the file into the text box and it will upload i have also sent you a message on discord to attach the debug information from SMM (Instructions on discord)

**Comment by Linrox** [comment]
Just says we don't support that type. game.sav

**Comment by Linrox** [comment]
I'll zip it first

**Comment by Linrox** [comment]
[SafeZone.zip](https://github.com/mrhid6/SF_Mod_RefinedPower/files/10001224/SafeZone.zip)

**Comment by Linrox** [comment]
The only other mod used was the Storage teleport, but i disabled it during testing

**Comment by Linrox** [comment]
Any updates on a fix for this issue?

**Comment by Linrox** [comment]
Guess you not going to fix the issue

**Comment by mrhid6** [comment]
Tested on Update 8 EXP.

After I loaded the Save file and removed all of the Refined R&D mods (RefinedPower, RefinedRDLib & ModularUI)
I can determine that the lag is still present.
This would indicate an issue with the save file or with SML itself.

I was getting around 19-30 FPS on a high-end PC and with settings all at LOW.

Removing mods from the game is not recommended, as this can leave memory pointers to invalid actors.
